R/matrix_form.R

```diff
@@ -212,7 +219,8 @@ MatrixPrintForm <- function(strings = NULL,
       prov_footer = prov_footer,
       col_gap = col_gap,
       table_inset = as.integer(table_inset),
-      has_topleft = has_topleft
+      has_topleft = has_topleft,
+      indent_size = indent_size
```

Comment:
I do not understand if this is good to be added here... I do not know if this is followed by an actual indentation somewhere before. Afterwards, it is for sure reconstructed as such